src/components/NotesPage/NotesPageSagas.js

```diff
@@ -0,0 +1,32 @@
+import { all, call, put, takeEvery } from 'redux-saga/dist/redux-saga-effects-npm-proxy.cjs';
+
+import {
+  CHANGE_HELP_VIEW_ACTION,
+  changeHelpViewAction,
+  changeSelectedNoteAction
+} from 'components/NotesPage/NotesPageActions';
+import thoughtStorage from 'storage/thought-storage';
+import { APP_INIT_SUCCESS } from 'components/App/AppActions';
+
+const HELP_VIEWED_KEY = 'controls_help:viewed';
+
+function* saveViewState({ data }) {
+  yield call([localStorage, localStorage.setItem], HELP_VIEWED_KEY, data);
+}
+
+function* readViewState() {
```

Comment:
maybe something like `readHelpSeenState` would sound better?

`readViewState()` sounds like nothing. "read state for a view". For which view? It's about whether the user has seen the help or not.